src/control/server/storage/scm/ipmctl.go

```diff
@@ -201,27 +201,24 @@ func (cr *cmdRunner) handleFreeCapacity(sockSelector int, nrNsPerSock uint, regi
 		return nil, nil, errors.Wrap(err, "createNamespaces")
 	}
 
-	numaSelector := sockAny
 	switch len(numaIDs) {
 	case 0:
-		return nil, nil, errors.New("no numa nodes were processed")
+		return nil, nil, errors.New("no namespaces created on regions with free capacity")
 	case 1:
```

Comment:
Nit: not sure the case 1 still needed.
If yes, the switch could probably be replace by a simple `if len(numaIDs) == 0`

Reply:
I think `case 1` is still needed, although it confused me for a minute, too. Maybe clearer:

```go
if len(numaIDs) == 0 {
    return return nil, nil, errors.New("no namespaces created on regions with free capacity")
}

if len(numaIDs) > 1 && sockSelector != sockAny {
    return nil, nil, errors.Errorf("unexpected number of numa nodes processed, want 1 got %d", len(numaIDs))
}
```

Reply:
I'm not clear why an empty switch statement is problematic, it is just allowing a drop through much like a break, I'm requesting reviews again because I don't see the need to change it.

Reply:
I won't block on it. It is technically correct. I just thought it might be worth changing if it confused two of us on the first read.

Personally I find it harder to parse and understand greater-than and less-than checks when using `switch` in this way, as opposed to either phrasing the cases as conditionals (e.g. `case len(numaIDs) > 1`), or using if-statements. I effectively had to translate the switch cases into conditionals for my own understanding.

Reply:
fixed